trendmicrovisionone_connector.py

```diff
@@ -835,850 +733,1438 @@ def _handle_remove_from_blocklist(self, param):
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def exception_list_count(self) -> int:
+        """Gets the count of objects present in exception list"""
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        new_exceptions: List[Dict[str, Any]] = []
+        try:
+            client.consume_exception_list(
+                lambda exception: new_exceptions.append(exception.dict())
+            )
+        except Exception as e:
+            self.debug_print("Consume Exception List failed with following exception:")
+            raise RuntimeError("Error while fetching exception list count.") from e
+
+        # Load json objects to list
+        exception_objects: List[Dict[str, Any]] = []
+        for i in new_exceptions:
+            i["description"] = "" if not i["description"] else i["description"]
+            i = json.dumps(i)
+            exception_objects.append(json.loads(i))
+        exception_count = len(exception_objects)
+        return exception_count
+
+    def _handle_add_to_exception(self, param):
+        """
+        Adds domains, file SHA-1, file SHA-256, IP addresses, sender addresses, or URLs to the Exception List.
+        Args:
+            block_objects(List[Dict[str, str]]): List of objects containing type, value and optional description.
+        Returns:
+            multi_resp(Dict[str, List]): Object containing task_id and HTTP status code.
+        """
+        self.save_progress(
+            "In action handler for: {0}".format(self.get_action_identifier())
+        )
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Required Params
+        block_objects: List[Dict[str, str]] = json.loads(param["block_objects"])
+
+        # Initialize Pytmv1
+        client = self._get_client()
+
+        multi_resp: List[MsData] = []
+
+        # Choose Enum
+        for i in block_objects:
+            obj_type = i["object_type"].upper()
+            if obj_type in ObjectType.__members__:
+                i["object_type"] = ObjectType[obj_type]
+            else:
+                raise RuntimeError(f"Please check object type: {i['object_type']}")
+
+        # Make rest call
+        for i in block_objects:
+            response = client.add_to_exception_list(
+                pytmv1.ObjectTask(
+                    object_type=i["object_type"],  # type: ignore
+                    object_value=i["object_value"],
+                    description=i.get("description", "Add To Exception List."),
+                )
+            )
+            if "error" in response.result_code.lower():
+                raise RuntimeError(
+                    f"Error while adding to exception list: {response.errors}"
+                )
+            assert response.response is not None
+            items = response.response.dict().get("items", [])[0]
```

Comment:
There is no need to use .dict() here when you can use response.response.items instead

Reply:
updated.